### PR TITLE
vdirsyncer 0.11.2

### PR DESCRIPTION
**Pull request by benjaminfrank**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/vdirsyncer.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
+# coding: utf-8
```
Comment (DomT4) [inline Formula/vdirsyncer.rb]: This still shouldn't be necessary. `utf-8` is set for all formulae files automatically.

Comment (benjaminfrank) [inline Formula/vdirsyncer.rb]: Sorry, emacs was quite insistent to add this line automagically. Should be fine now and I managed to convince my favourite editor to abide my commands in future ;)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/ea96878205791cc346c43dea2b353a1809f967c6. Thank you once again @benjaminfrank! 😺

File: Formula/vdirsyncer.rb (hunk 1 above)
Comment (DomT4) [inline Formula/vdirsyncer.rb]: No problem! Sometimes I wonder if we're controlling the machines or the machines are controlling us 😉.

**Pull request closed without merging**